src/pages/documentation/release-notes/index.md

```diff
@@ -128,14 +128,29 @@ import com.adobe.marketing.mobile.target.TargetParameters;
 
 ### Android Messaging 2.0.0
 
-* Major version update for [Adobe Journey Optimizer](../adobe-journey-optimizer/index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is already available as an [open source project on GitHub](https://github.com/adobe/aepsdk-messaging-android).
-* Adds support for Adobe Journey Optimizer powered in-app messages.
+* Major version update for [Adobe Journey Optimizer](../adobe-journey-optimizer/index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-messaging-android).
+
+### Android Places 2.0.0
+
+* Major version update for [Adobe Experience Platform Location Service](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-places-android).
+
+Please note that the following improvements have been made in the current release:
+
+1. `getNearbyPointsOfInterest` API without the errorcallback has been removed.
```

Comment:
Suggest rephrase:
The `getNearbyPointsOfInterest` API without the errorCallback has been removed. Alternatively, use the below overloaded API which provides both successCallback and errorCallback: 
```java
public static void getNearbyPointsOfInterest(final Location location,
			final int limit,
			final AdobeCallback<List<PlacesPOI>> successCallback,
			final AdobeCallback<PlacesRequestError> errorCallback)
```